packages/measure/README.md

```diff
@@ -0,0 +1,32 @@
+# Install
+
+Make sure that `pub` is on your path. Then run:
+installed.
+```shell
+pub global activate measure
+```
+
+# Run
+First, make sure that `dart` and `flutter` is available in your command line.
+
+Then, connect a **single** iPhone, run a Flutter app on it, and
+```shell
+# assuming that you're in this directory
+measure ioscpugpu new -u resources/TraceUtility -t resources/CpuGpuTemplate.tracetemplate
+```
+
+It currently outputs something like
```

Comment:
This language will become outdated and probably not get updated. Can we just say something like `Sample output:` and get rid of the whole `Eventually, we'd like to hook this...`?

Reply:
Done.